### ISSUE TITLE
[feature] make a node package to install via npm

### ISSUE THREAD

**Issue by caseykey** [ShivBhosale/React-Jupyter-Viewer#2]
I like this project but managing the dependency is cumbersome if it is not managed by node. Can we have this made into a node package? ie `npm install --save react-jupyter-viewer`?

**Comment by ShivBhosale** [comment]
Yes I've been planning to do this. I'll be committing a few more features before making it a node package however. The expected timeline for this is around Feb end

**Comment by caseykey** [comment]
Okay! I just recently published my own node package. I made a contribution and the author hadn't been merged it to the main project yet, but I needed the package on npm to deploy it to netlify. I followed [this](https://docs.npmjs.com/cli/publish) guide.  It was way easier than expected!

**Comment by ec2ainun** [comment]
any update on this case?
